src/demos/todomvc/todos.jsx

```diff
@@ -4,7 +4,10 @@ import TodosList from './todos-list';
 import TodoFooter from './todo-footer';
 import { ALL_TODOS, ACTIVE_TODOS, COMPLETED_TODOS } from './utils';
 
-import { AXADatepicker } from '../../js/react-exports';
```

Comment:
Since we added polyfill import to `index.js` and `react-exports`, they land twice on the page and innerSVG prolyfill throughs upon second attempt...